packages/core/admin/webpack.config.js

```diff
@@ -169,7 +169,12 @@ module.exports = ({
             },
           },
         },
-
+        {
```

Comment:
Can you add a comment please? In retrospective the commit message "Fix webpack" won't help much.